lldb/bindings/python/python-wrapper.swig

```diff
@@ -592,6 +592,8 @@ void *lldb_private::python::LLDBSWIGPython_CastPyObjectToSBExecutionContext(PyOb
   return sb_ptr;
 }
 
+#include "lldb/Interpreter/CommandReturnObject.h"
```

Comment:
Can we put this some place more reasonable (next to the other includes in `python.swig` ?)

Reply:
done